wikimedia/wikipedia-ios
# Image recommendations Bottom sheet feedback + display reason

**Phabricator:** Add reason https://phabricator.wikimedia.org/T360207
Feedback for https://phabricator.wikimedia.org/T356818

### Notes
* This PR handles most of the PR feedback for https://github.com/wikimedia/wikipedia-ios/pull/4777 with the exception of the crash
* The link button is now part of the text view, alongside the reason and description
* This PR also adds the reason for the suggestion to the bottom sheet

### Test Steps
1. Run the experimental app, click on the add an image explore feed card
2. Verify the layout still works, now with the reason being displayed
3. Take a look at other themes and RTL
